### PR TITLE
Seasons episodes

### PR DESCRIPTION
**Pull request by Sven-Lyco**
- Add SeasonsList to SeriesDetailsPage
- Add Episodes to SeasonsList
- Finish #7

### CODE REVIEW

**Review by macfrei — COMMENTED** [review]
Hello, nice job! Added a few comments you might want to consider 🙂, other than that, it looks great! 🚀

File: src/pages/SeriesDetailsPage.js [hunk 1]
```diff
@@ -69,14 +75,25 @@ export default function SeriesDetailsPage({
...
+                  : 'Aktuell ist leider keine Beschreibung verfügbar'}
+              </p>
+            </StyledMain>
+          ) : (
+            <>
+              <SeasonsList seriesId={id} seasons={seasons} />
+            </>
+          )}
         </>
       ) : (
```
[the remark this answers is not in the document]
Comment (Sven-Lyco) [reply src/pages/SeriesDetailsPage.js:98]: Thank you! 
I had not seen that yet and did not know that we did so :)

[the remark this answers is not in the document]
Comment (macfrei) [reply src/pages/SeriesDetailsPage.js]: You are welcome 👍

File: src/components/InnerNavigation.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import PAGES from '../assets/pages';
+
+export default function InnerNavigation({ currentPage, handleNavigation }) {
+  return (
+    <Wrapper>
+      <StyledDetailsButton currentPage={currentPage} onClick={handleNavigation}>
+        Details
+      </StyledDetailsButton>
+      <StyledSeasonButton currentPage={currentPage} onClick={handleNavigation}>
```
Comment (macfrei) [inline src/components/InnerNavigation.js:10]: I'd pass the page details to the `handleNavigation` function:
```suggestion
      <StyledDetailsButton currentPage={currentPage} onClick={() => handleNavigation('deatils')}>
        Details
      </StyledDetailsButton>
      <StyledSeasonButton currentPage={currentPage} onClick={() => handleNavigation('seasons')}>
```

File: src/pages/SeriesDetailsPage.js [hunk 3]
```diff
@@ -85,6 +104,12 @@ export default function SeriesDetailsPage({
       )}
     </Wrapper>
   );
+
+  function handleNavigation() {
+    currentPage === PAGES.DETAILS
+      ? setCurrentPage(PAGES.SEASONS)
+      : setCurrentPage(PAGES.DETAILS);
+  }
```
Comment (macfrei) [inline src/pages/SeriesDetailsPage.js:112]: And here add the `page` to your state :)
```suggestion
  function handleNavigation(page) {
    setCurrentPage(page)
  }
```

Comment (Sven-Lyco) [reply src/pages/SeriesDetailsPage.js:112]: Awesome! Thank you very much! 🚀

**Pull request merged**